.github/workflows/wheels.yml

```diff
@@ -0,0 +1,168 @@
+          brew unlink mpich && brew install openmpi
+          cmake --version
+          # Uninstall libomp for compatibility with issue #817
+          brew uninstall --ignore-dependencies libomp || echo "libomp doesn't exist"
+          echo "$(brew --prefix)/opt/cmake/bin:$(brew --prefix)/opt/flex/bin:$(brew --prefix)/opt/bison/bin" >> $GITHUB_PATH
+
+      - name: Install readline
+        if: runner.os == 'macOS'
+        run: |
+          sudo mkdir -p /opt/nrnwheel/$(uname -m)
+          sudo bash packaging/python/build_static_readline_osx.bash
+
+      - name: Set env for release
+        if: github.event_name == 'workflow_dispatch'
+        run: |
+          echo "NRN_NIGHTLY_UPLOAD=false" >> $GITHUB_ENV
+          echo NRN_RELEASE_UPLOAD=${{ github.event.inputs.upload }} >> $GITHUB_ENV
+          echo "NEURON_NIGHTLY_TAG=" >> $GITHUB_ENV
+          echo SETUPTOOLS_SCM_PRETEND_VERSION=${{ github.event.inputs.rel_version }} >> $GITHUB_ENV
+
+      - name: Build wheel
+        if: runner.os == 'macOS'
+        run: |
+          packaging/python/build_wheels.bash osx ${{ matrix.python-version }} coreneuron
+
+      - name: Build Manylinux wheel
+        if: runner.os != 'macOS'
+        run: |
+          docker run --rm \
+            -w /root/nrn \
+            -v $(pwd):/root/nrn \
+            -e NEURON_NIGHTLY_TAG \
+            -e NRN_NIGHTLY_UPLOAD \
+            -e NRN_RELEASE_UPLOAD \
+            -e SETUPTOOLS_SCM_PRETEND_VERSION \
+            -e NRN_BUILD_FOR_UPLOAD=1 \
+            ${{ matrix.base_image }} \
+            packaging/python/build_wheels.bash linux ${{ matrix.python-version }} coreneuron
+
+      - name: Upload wheel files
+        uses: actions/upload-artifact@v4
+        with:
+          name: wheels-${{ matrix.python-version }}-${{ matrix.os }}
+          path: wheelhouse/*.whl
+
+      - name: Setup Python ${{ matrix.python-version }} for testing
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+
+      - name: Install test dependencies
+        if: runner.os != 'macOS'
+        run: |
+          sudo apt update
+          sudo apt install -y mpich openmpi-bin libopenmpi-dev libmpich-dev
+
+      - name: Test wheel with ${{ matrix.python-version }}
+        run: |
+          minor_version="$(python${{ matrix.python-version }} -c 'import sys;print(sys.version_info.minor)')"
+          packaging/python/test_wheels.sh $(which python${{ matrix.python-version }}) wheelhouse/*cp3${minor_version}*.whl
+
+  merge:
+    name: Merge artifacts
+    runs-on: ubuntu-latest
+    needs: [build-test]
+    steps:
+      - name: Merge Artifacts
+        uses: actions/upload-artifact/merge@v4
+        with:
+          delete-merged: true
+          name: wheels
+          pattern: wheels-*
+
+  upload:
+    name: Upload wheels
+    runs-on: ubuntu-latest
+    needs: merge
+    steps:
+      - name: Upload wheels
+        run: echo "TODO"
```

Comment:
@JCGoran : Just mentioning - we need to update the docs from https://nrn.readthedocs.io/en/8.2.6/install/python_wheels.html#publishing-the-wheels-on-pypi-via-azure.
Especially, having the ability to just build the release artifacts without upload has been quite helpful.

Reply:
Right, there's also the matter of "Azure drop URLs" being mentioned in other repos, so we should make sure those are modified accordingly.